indexing-service/src/main/java/org/apache/druid/indexing/common/TaskToolbox.java

```diff
@@ -98,35 +98,36 @@
   private final DruidNode druidNode;
   private final LookupNodeService lookupNodeService;
   private final DataNodeService dataNodeService;
+  private final boolean useOak;
 
   public TaskToolbox(
-      TaskConfig config,
-      TaskActionClient taskActionClient,
-      ServiceEmitter emitter,
-      DataSegmentPusher segmentPusher,
-      DataSegmentKiller dataSegmentKiller,
-      DataSegmentMover dataSegmentMover,
-      DataSegmentArchiver dataSegmentArchiver,
-      DataSegmentAnnouncer segmentAnnouncer,
-      DataSegmentServerAnnouncer serverAnnouncer,
-      SegmentHandoffNotifierFactory handoffNotifierFactory,
-      Provider<QueryRunnerFactoryConglomerate> queryRunnerFactoryConglomerateProvider,
-      ExecutorService queryExecutorService,
-      MonitorScheduler monitorScheduler,
-      SegmentLoader segmentLoader,
-      ObjectMapper objectMapper,
-      File taskWorkDir,
-      IndexIO indexIO,
-      Cache cache,
-      CacheConfig cacheConfig,
-      CachePopulatorStats cachePopulatorStats,
-      IndexMergerV9 indexMergerV9,
-      DruidNodeAnnouncer druidNodeAnnouncer,
-      DruidNode druidNode,
-      LookupNodeService lookupNodeService,
-      DataNodeService dataNodeService,
-      TaskReportFileWriter taskReportFileWriter
-  )
+          TaskConfig config,
```

Comment:
probably an unintended change - worth to see which style format is used by druid devs

Reply:
Yup that's unintentional.

 Druid's maven build uses the checkstyle plugin to enforce their style (and their settings are quite strict).